addons/mail/static/src/components/emoji_grid_loading_screen/emoji_grid_loading_screen.scss

```diff
@@ -0,0 +1,7 @@
+.o_EmojiGridLoadingScreen {
+}
```

Comment:
Useless rule (basically whole SCSS file should be deleted, as other stuffs looks useless and/or replaced by classname)